docs/EBYTE_E77_MBL.md

```diff
@@ -0,0 +1,39 @@
+# mLRS Documentation: EBYTE E77 MBL Series Boards #
+
+([back to main page](../README.md))
+
+EBYTE MBL Series Boards are an option for building mLRS equipment. These boards use the EBYTE E77 module and are available in both 868/915 MHz and 433 MHz/70 cm versions. However, these boards are not perfect since their pins are not ready-made for the purposes of mLRS. So, some tweaking and (easy) soldering is required.
+
+## EBYTE E77 MBL Series Boards ##
+
+- [E77-400MBL-01 (Available on AliExpress)](https://www.cdebyte.com/products/E77-400MBL-01)
+- [E77-900MBL-01 (Available on AliExpress)](https://www.cdebyte.com/products/E77-900MBL-01)
+
+### As Tx Module ###
+
+Connections (name in respect to board print-ons, otherwise please refer to graphics):
+
+- serial: TxD,RxD and on-board USB plug
+- com/cli: TxD,RxD and on-board USB plug
+- debug: PA5
+- button1: BOOT
+- button2: cli / BIND
+
+Notes:
+
+- To enable the cli, hold down button2 during boot
+- To communicate with the radio via the JR bay (pin 5 in the JR bay), you need to solder a Schottky diode (e.g. BAT42) between pads PB7 and PB6 (PB7 - |<| - PB6, |<| represents the diode)
```

Comment:
in stead of the text in the (), maybe (see picture)?

Reply:
Updated note to 'see graphic'